File: cash.py
while True:
    n = input("Change owed: ")
    # http://qaru.site/questions/1283/how-do-i-check-if-a-string-is-a-number-float-in-python
    if n.replace(".", "", 1).isdigit():
        break
n = int(float(n) * 100)
quarter = 0;
if n >= 25:
    quarter = n // 25
dime = 0
if n >= 10:
    dime = (n - quarter * 25) // 10
nickel = 0
if n >= 5:
    nickel = (n - quarter * 25 - dime * 10) // 5
peny = n - quarter * 25 - dime * 10 - nickel * 5
print(quarter + dime + nickel + peny)

File: cash.c
#include <cs50.h>
#include <stdio.h>
#include <math.h>

int main(void)

{
    float f;
    int change = 0, quarter = 0, dime = 0, nickel = 0, penny = 0;

        // get change owened
    f = get_float("Change owed: ");

         while (f <= 0)
    {
        f = get_float("Change owed: ");
    }

    change = roundf (f*100);

    if (change >= 25)
    {
        quarter = quarter + (change-change%25)/25;
        change = change%25;
    }

    if (change >= 10)
    {
        dime = dime + (change - change%10)/10;
        change = change%10;
    }

    if (change >= 5)
    {
        nickel = nickel + (change - change%5)/5;
        change = change%5;
    }

    penny = change;
    change = quarter + dime + nickel + penny;
    printf ("%i\n", change);
}
